README.md

```diff
@@ -267,6 +268,25 @@ createUrl(urlPattern, {params, query});
 'https://api.com/users/1/teams/2?page=1&sort=name'
 ```
 
+##### createRouter(routes)
+Returns a `router` object for the given routes.
```

Comment:
Id like to make it more descriptive so people know why this function exists.

Creates a `router` object from a hash of named routes i.e. your sitemap. Named routes let you access the url patterns of your application all throughout your code by simply using a name and offload the actual url construction with parameters and query strings to the `router.get` method.